pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -386,3 +386,43 @@ func createDivModNPackedDivmodV1Hinter(resolver hintReferenceResolver) (hinter.H
 
 	return newDivModNPackedDivmodV1Hint(a, b), nil
 }
+
+// IsZeroDivMod hint computes the division modulo SECP_P prime for a given packed value
+//
+// `newIsZeroDivModHint` doesn't take any operander as argument
+//
+// `newIsZeroDivModHint` assigns the result as `x_inv` and `value` in the current scope
+func newIsZeroDivModHint() hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "IsZeroDivMod",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P
+			//> from starkware.python.math_utils import div_mod
+			//> value = x_inv = div_mod(1, x, SECP_P)
+
+			secpBig, _ := secp_utils.GetSecPBig()
+
+			x, err := ctx.ScopeManager.GetVariableValueAsBigInt("x")
+			if err != nil {
+				return err
+			}
+
+			resBig, err := secp_utils.Divmod(big.NewInt(1), x, &secpBig)
+			if err != nil {
+				return err
+			}
+
+			valueBig := new(big.Int).Set(&resBig)
+
+			if err := ctx.ScopeManager.AssignVariable("value", valueBig); err != nil {
```

Comment:
why make a new `valueBig`? cant we just use `resBig`?

Reply:
Hmm... I was earlier assigning x_inv also, Hence using resBig separately. Since I removed that, I will use resBig now for the only assignment. Thanks for pointing it out.